docs-v2/understand/concepts/actions.mdx

```diff
@@ -8,45 +8,61 @@ description: 'Learn more about actions.'
 
 Actions within Nango are designed to facilitate direct workflows with external APIs, such as creating a contact in Hubspot or sending a message via Slack. Unlike [syncs](/understand/concepts/syncs), actions require manual triggering and can provide immediate responses, making them ideally suited for dynamic, real-time interactions.
 
-# Use Cases for Actions
+# Use cases for actions
```

Comment:
imo it makes sense for it to be uppercase since it's Nango concept

Reply:
Agreed! Edited.